ArchiSteamFarm/Storage/GlobalConfig.cs

```diff
@@ -504,6 +504,16 @@ public sealed class GlobalConfig {
 				return (null, null);
 			}
 
+			if (globalConfig.IPCPasswordFormat == ArchiCryptoHelper.EHashingMethod.PlainText) {
+				ASF.ArchiLogger.LogGenericWarning(string.Format(CultureInfo.CurrentCulture, Strings.WarningPasswordSavedAsPlaintext, nameof(IPC)));
+
+				if (globalConfig.IsIPCPasswordSet && globalConfig.IPCPassword != null) {
+					if (Utilities.IsWeakPassword(globalConfig.IPCPassword, "ipc", "api", "gui", "asf-ui")) {
```

Comment:
Those words should have their own `const`, in this case like above `static readonly ImmutableHashSet<string>`. In fact I don't see why we shouldn't merge those with `ForbiddenPasswordWords`, all of those should apply cross-each-other.

Reply:
I've merged them and for "api", "ipc" and "asf-ui" this looks fine to me, but I'm not sure, we should disallow "gui" for Steam passwords. This one is quite easy to get through whatever logic someone uses to derive their password, since it is part of quite a lot of words.

Reply:
So just remove it, the same rule applies to IPC password, just because it has "gui" inside doesn't mean it's weak, according to you how it's used in a lot of words.